client/web/compose/src/components/PageBlocks/GeometryConfigurator/FeedSource/configs/Record.vue

```diff
@@ -28,16 +28,21 @@
           horizontal
           :label-cols="3"
           breakpoint="md"
-          :label="$t('geometry.recordFeed.colorLabel')"
+          :label="$t('geometry.recordFeed.geometryFieldLabel')"
         >
-          <b-input-group>
-            <b-form-input
-              v-model="feed.options.color"
-              style="max-width: 50px;"
-              type="color"
-              debounce="300"
-            />
-          </b-input-group>
+          <b-form-select
+            v-model="feed.geometryField"
```

Comment:
Here you can see the v-model is tied to the geometryField not the deeper options.geometryField